### DISCUSSION TITLE
[Feedback] Mention the potential need for --l1.trustrpc flag in op-node docs

### DISCUSSION THREAD

**Discussion by datanexus-vincent** [ethereum-optimism/developers#126] [Feedback]
### Feedback

I was having trouble getting a bedrock OP node up and running because I was using our own Erigon nodes as the L1 backend, and when you're doing that you apparently need to add the `--l1.trustrpc` flag to get it working.

That would probably be a good thing to mention in this section of the docs: https://docs.optimism.io/builders/node-operators/tutorials/mainnet#start-op-node-with-the-following-command

Thanks!

**Comment by sbvegan** [comment]
Thank you @datanexus-vincent, I've just flagged this internally. We'll work on getting this information in the docs!

**Comment by cpengilly** [comment]
agreed! issue can be tracked in docs repo here: https://github.com/ethereum-optimism/docs/issues/304

**Reply by sbvegan** [reply]
PR here: https://github.com/ethereum-optimism/docs/pull/312

**Comment by PierrickGT** [comment]
I had to use `--l1.trustrpc` flag too when using my own Nethermind node as the L1 backend to be able to sync in snap sync mode.
@sbvegan is this a known issue or I may have improperly configured my op-geth node?
